package.json

```diff
@@ -8,7 +8,10 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
+    "bootstrap": "^5.0.0-beta3",
     "core-js": "^3.6.5",
+    "sass": "^1.32.8",
+    "sass-loader": "^10.1.1",
```

Comment:
Warum jetzt doch sass? Hatten wir nicht gesagt, wir wollen keinen preprocessor nutzen? Ist meiner Meinung nach nicht nötig

Reply:
Ich vermute, du wolltest sass wegen variablen nutzen? Kann css nativ schon!

Reply:
Ja, hast eigentlich Recht. geht um die Bootstrap Variablen, siehe hier:
https://github.com/sebdei/hackercv/blob/colors/src/assets/styles/custom.scss

Reply:
Gehen würde das schon. Allerdings müsste man dann für jeden krams ein override in `custom.css` (Ich bevorzuge `bootstrap-override.css`) anlegen. Da würde es dann mit sass sicherlich eleganter sein.

Wenn dir das wichtig ist, nimms rein. Ich würde es draußen lassen.

Reply:
Ich hätte gerne die Möglichkeit, Farben zu überschreiben, also ja.